apps/src/sites/studio/pages/foorm/forms/editorHelpers.js

```diff
@@ -8,6 +11,29 @@ import _ from 'lodash';
 
 let codeMirror;
 
+const nameKeyRegex = new RegExp(/"(?:name|value)"\:\s*"(.+)",?/gi);
+const nameKeyValidator = new RegExp(/^[a-z0-9_]+$/i);
+
+// performs additional key validation
+// (non-basic characters can get stripped when being transferred to the server)
+export const lintFoormKeys = (text, options, cm) => {
+  const annotations = [];
+  const nameEntries = Array.from(text.matchAll(nameKeyRegex));
```

Comment:
oh, one thing i forgot to check is whether we polyfill matchAll (doesn't work in any version of IE)

Reply:
🤷 would that be apparent in transpiled version of this file?

Reply:
it was not being transpiled (and was broken in IE), didn't wanna try to get the polyfill to work, so i just switched it out for the equivalent code using `exec`. It's still broken in IE, but for different unrelated to this code reasons 😅 . I don't think anyone who interacts with this page uses IE anyways probably?

Reply:
equivalent code from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll